packages/shared/src/graphql-server/resolvers/profile.ts

```diff
@@ -3,13 +3,13 @@ import { getUserProfileAsync } from '../utils/getUserProfileAsync';
 import { setUserReferrerAsync } from '../utils/setUserReferrerAsync';
 
 export async function me(parent: unknown, args: Record<string, unknown>, context: ApolloContext): Promise<User | null> {
-  return await getUserProfileAsync(context);
+  return getUserProfileAsync(context);
```

Comment:
why remove await? I think we'll need to remove the async modifier on the line above if we do this.

Reply:
some combo of eslint or prettier rules actually did this automatically, it gave some error that await wasn't valid in that context